### ISSUE TITLE
importmap json does not include a package in package.json

### ISSUE THREAD

**Issue by phanhaiquang** [rails/importmap-rails#52]
Hi,

I am using "face-api.js" package (https://github.com/justadudewhohacks/face-api.js).

When I use "./bin/importmap pin face-api.js", there are packages added to config/importmap.rb and I have a problem with tslib because it does not provide default export. It seems that this is common issue with ES6 tslib.

So that I want to use package.json to manage this lib.
  yarn add face-api.js

Then, I "pin" this inside config/importmap.rb
  pin 'face-api.js'

Add it to application.js
  import "face-api.js";

And, use it in my js component
  import * as faceapi from "face-api.js";

After restart rails server, i got this error
---- error on my browser's console ----
   Uncaught TypeError: Failed to resolve module specifier "face-api.js". Relative references must start with either "/", "./", or "../".

Rails server reports some more warning in console
---- rails server ----
  Rendered logins/index.html.erb within layouts/application (Duration: 7.4ms | Allocations: 2177)
Importmap skipped missing path: face-api.js.js

I tried to change "face-api.js" to "face-api", but it does not work. 

Using "importmap json" does not show this package in my import map

I am not sure what is missing. Could you please give a support ?

Thanks,

**Comment by dhh** [comment]
Importmap is an alternative to yarn, not an addition to it. You use it instead of it.

bin/importmap pin face-api

**Comment by phanhaiquang** [comment]
> @phanhaiquang if you try clearing your browser cache the default export issue should be resolved now. Unfortunately @tensorflowjs/core doesn't currently seem to be working with another error - I've create a JSPM project issue to track this in [jspm/project#134](https://github.com/jspm/project/issues/134), you can watch that issue further.

Thank @guybedford . I clear cache and the error disappeared. But it faces another error as you mention.  :(

**Comment by phanhaiquang** [comment]
@dhh I saw your YouTube and you mention that we could also use importmap & yarn together. It is the way you added "trix" to package.json, and pin it in config/importmap.rb.

I tried to do the same by adding the lib to package.json and pin it in config file. But I am not sure what is still missing. Do you mean that I could not pin a package inside node_modules ? And if I need to use the lib, I need to download the lib and put to "vendor" folder ?

**Comment by phanhaiquang** [comment]
Thanks a lot @dhh for prompt response !

**Comment by phanhaiquang** [comment]
I used ES6 for that "importmap pin" and it has error. Googling shows that the error is not completely resolved (another error happens after i ckean my cached)

At that time, I did not try to change the provider from ES6 to NPM (actually, I did not know that I can change. Because I did not read your readme carefull).

**Comment by Hunter2065** [comment]
> 

No, for my bad it just returns `Couldn't find any packages in ["face-api.js"] on jspm`
